study.md

```diff
@@ -151,7 +160,10 @@ What are the 4 most common HTTP verbs used when creating a RESTful API. How
 would you use each?
 
 ```md
-<!-- your answer here -->
+GET -  used to retrieve info
+PUT - used to place information on the server such as a file
+POST - used to process information on the server such as a form submission
+DELETE - mostly self explanatory, but....it deletes things - opposite of PUT
```

Comment:
so POST is actually the opposite of DELETE, POST will create a new resource where as PUT and PATCH are similar in that they modify an existing resource.